test/e2e/federation-apiserver.go

```diff
@@ -84,8 +84,44 @@ var _ = framework.KubeDescribe("Federation apiserver [Feature:Federation]", func
 			framework.Logf("Verified that zero clusters remain")
 		})
 	})
+	Describe("Admission control", func() {
+		AfterEach(func() {
+			framework.SkipUnlessFederated(f.Client)
+		})
+
+		It("should not be able to create resources if namespace does not exist", func() {
+			framework.SkipUnlessFederated(f.Client)
```

Comment:
Sorry did not get you. Do you mean move SkipUnlessFederated to BeforeEach?
We put that line in BeforeEach, AfterEach and It() - all 3.